client/verta/verta/tracking/entities/_deployable_entity.py

```diff
@@ -88,6 +90,74 @@ def _get_artifact_msg(self, key):
     def _get_artifact(self, key):
         raise NotImplementedError
 
+    def set_schema(self, input: dict, output: dict = None) -> None:
+        """
+        Sets the input and output schemas of this Deployable Entity. Schemas are stored as
```

Comment:
Since "Deployable Entity" is somewhat of an internal construct customers won't recognize, perhaps we could just say "Sets the input and output schemas, which are stored ..." and leave it at that?

Reply:
I was kind of thinking that too